pkg/cvo/internal/generic.go

```diff
@@ -28,6 +29,9 @@ func readUnstructuredV1OrDie(objBytes []byte) *unstructured.Unstructured {
 }
 
 func applyUnstructured(client dynamic.ResourceInterface, required *unstructured.Unstructured) (*unstructured.Unstructured, bool, error) {
+	if required.GetName() == "" {
+		return nil, false, fmt.Errorf("invalid object: name cannot be empty")
```

Comment:
hrm.  Is there some way we can get an identifier here to help the log-reader figure out what needs fixing?  Ordinarily that would be the name, but in this case... ?  Maybe push this up into `genericBuilder.Do` where we can dump the broken `b.raw`?

Reply:
Or maybe keep more information from the initial `m lib.Manifest` in `NewGenericBuilder`?

Reply:
https://github.com/openshift/cluster-version-operator/blob/435610134231fae903b6e57f6ea0d8f9c1511a68/pkg/cvo/sync.go#L51

We print out the details of the task.